internal/pkg/workspace/workspace.go

```diff
@@ -125,6 +125,21 @@ func (ws *Service) writeSummary(projectName string) error {
 	return ws.fsUtils.WriteFile(summaryPath, serializedWorkspaceSummary, 0644)
 }
 
+//LocalApps returns the name of all the local apps
+func (ws *Service) LocalApps() ([]string, error) {
+	manifests, err := ws.ListManifestFiles()
+	if err != nil {
+		return nil, err
+	}
+	var apps []string
+	for _, manifest := range manifests {
+		appFile := filepath.Base(manifest)
+		appName := appFile[0 : len(appFile)-len(manifestFileSuffix)]
+		apps = append(apps, appName)
```

Comment:
Is it safe to extract the app names from the file names? I think so? Can we document that assumption if that's the case please?

Reply:
I think the safest option would be reading the "name" field from the manifests themselves.